tests/pass-dep/tokio_mvp.rs

```diff
@@ -0,0 +1,6 @@
+//@compile-flags: -Zmiri-disable-isolation -Zmiri-permissive-provenance
+//@only-target-linux: support for tokio only on linux and x86
+//@only-target-x86_64: see above
```

Comment:
```suggestion
//@only-target-x86_64-unknown-linux: we support epoll only on linux and x86
```
might as well combine the filters then

Reply:
This doesn't actually work out of some reason.

Reply:
Yeah, that's [this bug](https://github.com/oli-obk/ui_test/issues/24#issuecomment-1297505573).